gemfire-sink/README.adoc

```diff
@@ -0,0 +1,37 @@
+= Spring Cloud Stream Gemfire Sink
+
+A *Spring Cloud Stream* module that allows one to write message payloads to a Gemfire server.
+
+== Requirements
+
+* Java 7 or above
+* Gemfire 8.0 or above 
+
+Note: This module uses Gemfire v8.0.0. To use with earlier versions of Gemfire, you must change the Gemfire versions in 
+the root pom file and rebuild.
+
+== Code Tour
```

Comment:
I believe the latest template for section titles is just "Classes", and "Building", "Running" below. cc @markfisher